uint32 lost_samples_count
bool diag_checksum_error_flag
bool diag_flash_memory_write_count_exceeded_error
bool diag_aduc_mcu_fault
bool diag_x_axis_accelerometer_failure
bool diag_y_axis_accelerometer_failure
bool diag_z_axis_accelerometer_failure
bool diag_x_axis_gyroscope_failure
bool diag_y_axis_gyroscope_failure
bool diag_z_axis_gyroscope_failure
bool diag_sensor_initialization_failure
bool diag_clk_error
bool diag_flash_memory_test_error
bool diag_sensor_self_test_error
bool diag_standby_mode
bool diag_spi_communication_error
bool diag_flash_memory_update_error
bool diag_data_path_overrun
uint32 flash_counter
